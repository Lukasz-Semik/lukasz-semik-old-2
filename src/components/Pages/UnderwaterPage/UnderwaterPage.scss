.wrapper {
  position: relative;
  font-family: 'Caveat Brush', cursive;
}

.title-wrapper {
  @include absolute-center;
}
