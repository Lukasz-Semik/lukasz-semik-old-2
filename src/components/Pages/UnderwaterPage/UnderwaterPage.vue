<template>
  <div :class="[$style['wrapper']]">
    <background-element/>

    <div :class="[$style['title-wrapper']]">
      <wavy-title/>
    </div>
  </div>
</template>

<script>
import { BackgroundElement, WavyTitle } from '@/components/Atoms';

export default {
  name: 'UnderwaterPage',
  components: {
    BackgroundElement,
    WavyTitle,
  },
};
</script>

<style lang="scss" module src="./UnderwaterPage.scss" />
