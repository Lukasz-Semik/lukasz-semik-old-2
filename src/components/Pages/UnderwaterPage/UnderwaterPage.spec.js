import { shallowMount } from '@vue/test-utils';

import UnderwaterPage from './UnderwaterPage.vue';

describe('<UnderwaterPage />', () => {
  it('should render UnderwaterPage properly', () => {
    const wrapper = shallowMount(UnderwaterPage);

    expect(wrapper).toMatchSnapshot();
  });
});
