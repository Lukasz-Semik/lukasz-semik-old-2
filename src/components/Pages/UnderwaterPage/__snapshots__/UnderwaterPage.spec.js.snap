// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UnderwaterPage /> should render UnderwaterPage properly 1`] = `
<div class="wrapper">
  <background-element-stub></background-element-stub>
  <div class="title-wrapper">
    <wavy-title-stub translationpath="underwater.welcomeText"></wavy-title-stub>
  </div>
</div>
`;
